electron-app/src/components/inputs/AssetSelect.vue

```diff
@@ -60,6 +70,19 @@ export default class AssetSelect extends Vue {
   @Emit()
   input(_value: string) {}
 
+  get assets(): SupportedAsset[] {
+    if (this.items) {
+      const filteredAssets = this.supportedAssets.filter(asset => {
+        if (this.items!.find(item => item === asset.key)) {
+          return asset;
+        }
+      });
+      return filteredAssets;
```

Comment:
```suggestion
   return this.supportedAssets.filter(asset => this.items.includes(asset.key))
```
Instead of find you can use `includes` in the filter

Reply:
Nice tip, thanks!